pipeline-model-definition/src/test/java/org/jenkinsci/plugins/pipeline/modeldefinition/LibrariesTest.java

```diff
@@ -161,6 +161,8 @@ public void librariesDirectiveWithOutsideVarAndFunc() throws Exception {
     @Test
     public void folderLibraryParsing() throws Exception {
         otherRepo.init();
+        // longpaths to support longer filenames for windows. Only needs to be set once.
+        otherRepo.git("config", "--global", "core.longpaths", "true");
```

Comment:
Can be reverted now that https://github.com/jenkins-infra/pipeline-library/pull/334 is merged?

Reply:
Filed #506